src/components/TextEditor/components/StatusBar.js

```diff
@@ -1,10 +1,15 @@
-import React from "react";
+import React, { useState } from "react";
 
 const StatusBar = (props) => {
+  const edit = props.editing;
+  const textStyle = { color: "white" };
+  if (props.theme === "light") {
+    textStyle.color = "black";
+  }
   return (
     <div className="status-bar-container">
-      <p className="status-bar-info">
-        {props.editing ? "Editing" : "Viewing"} line: {props.line + 1} col: {props.col + 1}{" "}
+      <p className="status-bar-info" style={textStyle}>
```

Comment:
I still think that we could integrate this with our built-in styles with something like this instead:
```
<p className="status-bar-info" className={props.theme === "light" ? "light-status" : "dark-status"}>
```